apache-exporter.yaml

```
@@ -0,0 +1,97 @@
+package:
+  name: apache-exporter
+  version: "1.0.10"
+  epoch: 0
+  description: apache-exporter exposes Apache server metrics for Prometheus monitoring
+  copyright:
+    - license: MIT
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/Lusitaniae/apache_exporter
+      tag: v${{package.version}}
+      expected-commit: c4206b529038d79f20f2eeba98a9780c8cb4a2c2
+
+  - uses: go/bump
+    with:
+      deps: |-
+        golang.org/x/crypto@v0.35.0
+        golang.org/x/net@v0.38.0
+        golang.org/x/oauth2@v0.27.0
+
+  - uses: go/build
+    with:
+      packages: .
+      output: apache_exporter
+      ldflags: |
+        -X github.com/prometheus/common/version.Version=${{package.version}} \
+        -X github.com/prometheus/common/version.Revision=$(git rev-parse --short HEAD) \
+        -X github.com/prometheus/common/version.Branch=$(git rev-parse --abbrev-ref HEAD) \
+        -X github.com/prometheus/common/version.BuildUser=$(whoami)@$(hostname) \
```

```suggestion
        -X github.com/prometheus/common/version.BuildUser=linky@chainguard.dev \
```
Lets use something static here as these can change between runners